TechnologyAdvice/semantic-ta
# Proper dev server

### Pivotal Items
- [x] :star: [add dev server that reloads on dist changes only](https://www.pivotaltracker.com/story/show/98152748)
- [x] :beetle: [global variables/overrides changes do not rebuild all affected components](https://www.pivotaltracker.com/story/show/98496410)
- [x] :beetle: [changes to docs/src do not trigger rebuilds and reloads](https://www.pivotaltracker.com/story/show/98503414)
#### Heads Up
- [x] Remove live-server
- [x] Split build and docs task
- [x] Move index.html to `docs/src`
#### How to test
1. Run gulp.
2. Change a component's variables/overrides file
   - See a _single component_ rebuild
   - Expect browser reload
3. Change a `/src/site/global/*` file
   - See a _complete_ rebuild
   - Expect browser reload
4. Change a `docs/src` file.
   - See a _docs_ build
   - Expect browser reload.
